docs/editor_manual/managing_redirects.md

```diff
@@ -24,8 +24,6 @@ As a best practice Wagtail will check redirects as permanent by default, in orde
 
 ## Configuring redirects
 
-![](../_static/images/screen41_redirects_menu_item.png)
```

Comment:
Removed because it just shows the menu item in the sidebar, which doesn’t feel very useful.